api/voice-react-native-sdk.api.md

```diff
@@ -172,10 +172,11 @@ export class Call extends EventEmitter {
     // Warning: (ae-forgotten-export) The symbol "NativeCallInfo" needs to be exported by the entry point index.d.ts
     //
     // @internal
-    constructor({ uuid, customParameters, from, sid, to, isMuted, isOnHold, }: NativeCallInfo);
+    constructor({ uuid, customParameters, from, sid, state, to, isMuted, isOnHold, initialConnectedTimestamp, }: NativeCallInfo);
     disconnect(): Promise<void>;
     getCustomParameters(): CustomParameters;
     getFrom(): string | undefined;
+    getInitialConnectedTimestamp(): number | undefined;

```

Comment:
This is a new public API right? This is not too big of an example, but I think we need to get into the habit of following our process for any new public APIs. If the changes requires new/updated API:
- Create/update IDL and get it reviewed, before implementing into the SDK
- Check other voice SDKs answer questions like
  - Is this applicable to other voice sdks?
  - Do we need to add similar feature?
  - How is this done on other voice sdks, if applicable?
- Probably not applicable in this PR but also part of the process. If the new feature or API is complex enough
  - Create a design doc
  - Review with the team